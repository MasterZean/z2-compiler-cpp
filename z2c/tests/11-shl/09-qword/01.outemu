QWord{1120}
QWord{1120}
QWord{1120}
QWord{1120}
QWord{1120}
QWord{1120}
QWord{1120}
QWord{1120}
QWord{1120}
