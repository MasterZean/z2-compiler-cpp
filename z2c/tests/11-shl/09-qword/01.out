70ull << 4;
70ull << 4u;
70ull << 4;
70ull << 4u;
70ull << 4;
70ull << 4u;
70ull << 4ll;
70ull << 4ull;
70ull << 4u;
DummyClass(1, 6): error:
	can't apply operator '<<' on types: 
		'QWord' and 
		'Bool'
DummyClass(2, 6): error:
	can't apply operator '<<' on types: 
		'QWord' and 
		'Bool'
DummyClass(11, 6): error:
	can't apply operator '<<' on types: 
		'QWord' and 
		'Float'
DummyClass(12, 6): error:
	can't apply operator '<<' on types: 
		'QWord' and 
		'Double'
DummyClass(13, 6): error:
	can't apply operator '<<' on types: 
		'QWord' and 
		'Char'
